pkg/auth/authenticator/password/keystonepassword/keystonepassword_test.go

```diff
@@ -24,50 +24,20 @@ func TestKeystoneLogin(t *testing.T) {
 
 	const ID = "0123456789"
 
-	th.Mux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
when we switched to AuthenticateV3, it doesn't need the version listing any more